### PR TITLE
[AUR-181] CBI integration tests

### PR DESCRIPTION
**Pull request by Ficzer**

### CODE REVIEW

File: requirements/AUR-CBI-INT-0010/component-management-behaviors/Makefile [hunk 1]
```diff
@@ -0,0 +1,136 @@
+KAZOO?=kazoo
```
Comment (Lurkerpas) [inline requirements/AUR-CBI-INT-0010/component-management-behaviors/Makefile:1]: This is applicable not to the file, but to the requirement. The validation method is M and I. M is satisfied. I do not see the expected test file with Inspection results.

Comment (Ficzer) [reply requirements/AUR-CBI-INT-0010/component-management-behaviors/Makefile:1]: Done, added in .md file

File: requirements/AUR-CBI-INT-0010/interface-view-cbi-extension.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# CBI model integration using SpaceCreator
+
+**Overview:**
+
+SpaceCreator was used as base for CBI model editior implementation as per [1]
```
Comment (Lurkerpas) [inline requirements/AUR-CBI-INT-0010/interface-view-cbi-extension.md:5]: Let's discuss this

Comment (Ficzer) [reply requirements/AUR-CBI-INT-0010/interface-view-cbi-extension.md:5]: Done

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**